src/usr/bin/diffuse

```diff
@@ -6817,15 +6826,16 @@ class NotebookTab(Gtk.EventBox):
     def __init__(self, name, stock):
         Gtk.EventBox.__init__(self)
         self.set_visible_window(False)
-        hbox = Gtk.HBox.new(False, 0)
+        hbox = Gtk.Box.new(Gtk.Orientation.HORIZONTAL, 0)
         if stock is not None:
             image = Gtk.Image.new()
             image.set_from_stock(stock, Gtk.IconSize.MENU)
             hbox.pack_start(image, False, False, 5)
             image.show()
         self.label = label = Gtk.Label.new(name)
         # left justify the widget
-        label.set_alignment(0, 0.5)
+        label.set_xalign(0.0)
+        label.set_yalign(0.5)
```

Comment:
I don't know if this is better GTK code, but I do think it is closer to what we had before ;)